wavesrv/pkg/sstore/sstore.go

```diff
@@ -1310,3 +1319,12 @@ func SetClientOpts(ctx context.Context, clientOpts ClientOptsType) error {
 	})
 	return txErr
 }
+
+func SetReleaseInfo(ctx context.Context, releaseInfo ReleaseInfoType) error {
+	txErr := WithTx(ctx, func(tx *TxWrap) error {
+		query := `UPDATE client SET releaseinfo = ?`
```

Comment:
you're going to need to use quickJson() here to convert releaseinfo to a string for sqlite